tests/Geocoder/Tests/Provider/GoogleMapsTest.php

```diff
@@ -196,60 +196,61 @@ public function testGeocodeBoundsWithRealAddressForNonRooftopLocation()
     public function testGeocodeWithRealAddressReturnsMultipleResults()
     {
         $provider = new GoogleMaps($this->getAdapter());
-        $results  = $provider->geocode('Paris');
+        $results  = $provider->geocode('i');
```

Comment:
This line should not be changed. Keep "Paris" here and we do not have to modify the rest of the test.